pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tortoise-orm"
-version = "0.21.7"
```

Comment:
Please don't change it for now - I am planning to release it as 0.22.0 - after PR with table name generation